build_runner/README.md

```diff
@@ -99,14 +100,71 @@ directories are served, and on what ports.
 For example to serve the `example` and `web` directories on ports 8000 and 8001
 you would do `dart run build_runner serve example:8000 web:8001`.
 
-##### test
+### test
 
 The test command will forward any arguments after an empty `--` arg to the
 `dart run test` command.
 
 For example if you wanted to pass `-p chrome` you would do
 `dart run build_runner test -- -p chrome`.
 
+### build-filter
+Build filters allow you to choose explicitly which files to build instead of building entire directories.
+
+A build filter is a combination of a package and a path, with glob syntax supported for each.
+
+Whenever a build filter is provided, only required outputs matching one of the build filters will be built, in addition to the inputs to those outputs.
+
+##### Command Line Usage:
+
+Build filters are supplied using the new `--build-filter` option, which accepts relative paths under the package as well as `package:` uris.
+
+Glob syntax is allowed in both package names and paths.
+
+*Example*: The following would build and serve the JS output for an application, as well as copy over the required SDK resources for that app:
+
+```bash
+pub run build_runner serve \
```

Comment:
```suggestion
dart run build_runner serve \
```

Reply:
Sure thing, I'll make the change. @jakemac53 I copied this from https://pub.dev/packages/build_runner/changelog#170 but don't have that file in the repo to change.